x-pack/filebeat/module/okta/_meta/docs.md

```diff
@@ -0,0 +1,100 @@
+:::::{admonition} Prefer to use {{agent}} for this use case?
+Refer to the [Elastic Integrations documentation](integration-docs://reference/okta/index.md).
+
+::::{dropdown} Learn more
+{{agent}} is a single, unified way to add monitoring for logs, metrics, and other types of data to a host. It can also protect hosts from security threats, query data from operating systems, forward data from remote services or hardware, and more. Refer to the documentation for a detailed [comparison of {{beats}} and {{agent}}](docs-content://reference/fleet/index.md).
+
+::::
+
+
+:::::
+
+
+The Okta module collects events from the [Okta API](https://developer.okta.com/docs/reference/). Specifically this supports reading from the [Okta System Log API](https://developer.okta.com/docs/reference/api/system-log/).
+
+
+### Variable settings [okta-settings]
+
+Each fileset has separate variable settings for configuring the behavior of the module. If you don’t specify variable settings, the `okta` module uses the defaults.
+
+For advanced use cases, you can also override input settings. See [Override input settings](/reference/filebeat/advanced-settings.md).
+
+::::{tip}
+When you specify a setting at the command line, remember to prefix the setting with the module name, for example, `okta.system.var.paths` instead of `system.var.paths`.
+::::
+
+
+
+### `system` fileset settings [_system_fileset_settings]
+
+The Okta System Log records system events related to your organization in order to provide an audit trail that can be used to understand platform activity and to diagnose problems. This module is implemented using the [httpjson](/reference/filebeat/filebeat-input-httpjson.md) input and is configured to paginate through the logs while honoring any [rate-limiting](https://developer.okta.com/docs/reference/rate-limits/) headers sent by Okta.
+
+This is an example configuration for the module.
+
+```yaml
+- module: okta
+  system:
+    var.url: https://yourOktaDomain/api/v1/logs
+    var.api_key: '00QCjAl4MlV-WPXM...0HmjFx-vbGua'
+```
+
+
+#### Configuration options [_configuration_options_44]
+
+**`var.url`**
+:   Specifies the URL to the Okta System Log API. Required.
+
+    ```yaml
+        var.url: https://mycompany.okta.com/api/v1/logs
+    ```
```

Comment:
Confirming that code blocks like this are indented when they are examples for a configuration option. Is that correct?